packaging/docker/README.md

```diff
@@ -403,19 +403,20 @@ proxy.
 ```yaml
 version: '3'
 volumes:
-  caddy:
+  caddy_data:
+    external: true
```

Comment:
Can you elaborate why `caddy_data` is external?

According to the [docs](https://docs.docker.com/storage/volumes/#use-a-volume-with-docker-compose):

> A volume may be created directly outside of compose with docker volume create

Reply:
The intention when writing that was to avoid loss of data on `docker compose down`. I realize now that's wrong. Sorry, that was based on misunderstanding of the [docs](https://docs.docker.com/engine/reference/commandline/compose_down/#extended-description). Reading it carefully now, it says only anonymous volumes are removed, but not named nor host volumes.

Do you prefer keeping the `external: true` part or put a clarification in the docs?

Reply:
Yes, the default is to not remove named volumes.

>     -v, --volumes          Remove named volumes declared in the `volumes`
>                            section of the Compose file and anonymous volumes
>                            attached to containers.

I still see no reason for having `external: true`, I suggest removing it.

Reply:
Done.